libdnf/repo/solv_repo.cpp

```diff
@@ -93,12 +94,14 @@ namespace libdnf::repo {
 
 static const char * repodata_type_to_suffix(RepodataType type) {
     switch (type) {
-        case RepodataType::FILENAMES:
-            return "-filenames";
+        case RepodataType::FILELISTS:
+            return "-filelists";
         case RepodataType::PRESTO:
             return "-presto";
         case RepodataType::UPDATEINFO:
             return "-updateinfo";
+        case RepodataType::COMPS:
+            return "-comps";
```

Comment:
Isn't `-groups` also comps?

Reply:
Could you be more specific about your point?

Reply:
Hmm, I guess I was mistaken. For some reason, I thought `groups.xml` existed as reference to comps groups, but it doesn't.

Reply:
Okay, this suffix is also just for the libsolv cache file, so I don't think it would be related...